packages/integration-karma/test/dom-manual/moving-node-to-document/index.spec.js

```diff
@@ -0,0 +1,80 @@
+import { createElement } from 'lwc';
+import Container from 'x/container';
+
+describe('Moving elements from inside lwc:dom=manual', () => {
+    it('should return correct parentNode', function() {
```

Comment:
nitpick. All tests so far use an arrow function, why change the style now?
```suggestion
    it('should return correct parentNode', () => {
```